### PR TITLE
Still working app

### PR DESCRIPTION
**Pull request by asystentRoberta**

### CODE REVIEW

**Comment by asystentRoberta** [comment]
I wonder if I can merge pull request when reviev requested is set?

File: README.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ Tu [> do obejrzenia.](https://www.youtube.com/playlist?list=PLU2dl_1LV_STR4IV60K
 - [x] 07 Mapowanie zapytań: ścieżki i metody http 10:14
 - [x] 08 Zmienne z adresów URL 14:42
 - [x] 09 Parametryzowanie ścieżek 6:48
-- [ ] 10 Dopasowywanie do ścieżki z użyciem wyrażeń regularnych 6:15
-- [ ] 11 Przekazywanie parametrów do zapytania 11:24
+- [x] 10 Dopasowywanie do ścieżki z użyciem wyrażeń regularnych 6:15
+- [x] 11 Przekazywanie parametrów do zapytania 11:24
```
Comment (qrczepieczone) [inline README.md]: Sure 😄

File: src/main/java/pl/com/bohdziewicz/tutorials/mvcspring/CompanyController.java [hunk 2]
```diff
@@ -35,7 +35,7 @@ private Company findOne(
         return companyRepository.findOne(companyName);
     }
 
-    @GetMapping("{companyName}/{firstName}")
+    @GetMapping("{companyName}/${urls.company.employees.test}/{firstName:[a-zA-Z]+}")
```
Comment (qrczepieczone) [inline src/main/java/pl/com/bohdziewicz/tutorials/mvcspring/CompanyController.java]: You should change 
```java
urls.company.employees.test
```
to 
```java
urls.company.employees.root
```
Here and every place where you used 'test'

File: src/main/java/pl/com/bohdziewicz/tutorials/mvcspring/CompanyController.java [hunk 3]
```diff
@@ -58,4 +58,17 @@ private Company findOne(
                 .filter(employee -> Objects.equals(employee.getLastName(), pathVariable.get("lastName")))
                 .collect(Collectors.toList());
     }
+
+    @GetMapping("{companyName}/${urls.company.employees.test}/{employeeId:\\d+}")
```
Comment (qrczepieczone) [inline src/main/java/pl/com/bohdziewicz/tutorials/mvcspring/CompanyController.java]:
```suggestion
    @GetMapping("{companyName}/${urls.company.employees.root}/{employeeId:\\d+}")
```

File: src/main/java/pl/com/bohdziewicz/tutorials/mvcspring/EmployeesController.java [hunk 4]
```diff
@@ -4,28 +4,55 @@
...
+
+    public EmployeesController(CompanyRepository companyRepository) {
+
+        this.companyRepository = companyRepository;
+    }
+
     @RequestMapping("getEmployees")
     public List<Employee> getEmployees() {
 
         List<Employee> result = new ArrayList<>();
```
Comment (qrczepieczone) [inline src/main/java/pl/com/bohdziewicz/tutorials/mvcspring/EmployeesController.java]: Should be deleted (it is fake employees implementation)

File: README.md (hunk 1 above)
Comment (asystentRoberta) [reply README.md]: :ok:

File: src/main/java/pl/com/bohdziewicz/tutorials/mvcspring/CompanyController.java (hunk 2 above)
Comment (asystentRoberta) [reply src/main/java/pl/com/bohdziewicz/tutorials/mvcspring/CompanyController.java]: I will do later.

File: src/main/java/pl/com/bohdziewicz/tutorials/mvcspring/CompanyController.java (hunk 3 above)
Comment (asystentRoberta) [reply src/main/java/pl/com/bohdziewicz/tutorials/mvcspring/CompanyController.java]: Not now

**Comment by asystentRoberta** [comment]
Ok

**Pull request merged**